Tavisca.Bootcamp.LanguageBasics.Exercise1/Assignment1.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+
+namespace Tavisca.Bootcamp.LanguageBasics.Exercise1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Test("42*47=1?74", 9);
+            Test("4?*47=1974", 2);
+            Test("42*?7=1974", 4);
+            Test("42*?47=1974", -1);
+            Test("2*12?=247", -1);
+            Console.ReadKey(true);
+        }
+
+        private static void Test(string args, int expected)
+        {
+            var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
+            Console.WriteLine($"{args} : {result}");
+        }
+
+        public static int FindDigit(string equation)
+        {     
+            string first_num = equation.Split('*')[0];
+            string second_num = equation.Split('*')[1].Split('=')[0];

```

Comment:
use array of '*' and '=' to split string in a single operation and store in array. this array[0], [1] and [2] will give you a, b and c